content/reference/rest-api/templates/list.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Templates: list'
+weight: 10
+
+menu:
+  main:
+    name: 'list'
+    identifier: 'rest-api-templates-list'
+    parent: 'rest-api-templates'
+    pre: 'Lists templates.'
+---
+
+Lists templates with optional query parameters for filtering.
```

Comment:
I could imagine that by default only published templates are returned and we offer a parameter to return the unpublished version of a template. This could also apply to the get endpoint.